### PR TITLE
graph break on  sparse tensors constructions

### PR DESCRIPTION
**Pull request by laithsakka**
Fix some tests in https://github.com/pytorch/pytorch/issues/119780
sparse_bsc_tensor is not supported 
https://github.com/pytorch/pytorch/pull/117907

Also more about the issue here. 
https://docs.google.com/document/d/1EIb4qG88-SjVFn5TloLERliYdxIu2hwYoAA8skjOVfo/edit

Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #120458

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang @aakhundov

### CODE REVIEW

**Comment by ezyang** [comment]
Adding @yanboliang but I think you are supposed to do something in the dynamo allowed module instead

File: torch/_dynamo/variables/torch.py [hunk 1]
```diff
@@ -409,6 +409,8 @@ def call_function(
                 ),
                 example_value=None,
             )
+        elif self.value.__name__ == 'sparse_bsc_tensor':
+            unimplemented("sparse_bsc_tensor")
```
Comment (yanboliang) [inline torch/_dynamo/variables/torch.py:413]: The correct way to to this is:
1, Add ```"torch.sparse_bsc_tensor": SkipFunctionVariable,``` to ```manual_torch_name_rule_map``` in ```trace_rules.py```;
2, Remove ```"torch.sparse_bsc_tensor"``` from ```torch_c_binding_in_graph_functions``` in ```trace_rules.py```.
And there are a few similar functions, probably you want have the same treatment:
https://github.com/pytorch/pytorch/blob/2240018c03744ee34ea14ad53481db934c37e384/torch/_dynamo/trace_rules.py#L1989-L1994

Comment (laithsakka) [reply torch/_dynamo/variables/torch.py:413]: thanks for the tip

**Review by ezyang — APPROVED** [review]
okey dokey

**Comment by laithsakka** [comment]
I think i expect more tests to be unblocked will take a quick other look on some tests before landing

**Comment by laithsakka** [comment]
ahh i see could be of the pending, idk what they have been pending since yesterday though!?

**Comment by laithsakka** [comment]
remove on more unexpected success

**Pull request closed without merging**